clock_exercise/alarmitemdata.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <QObject>
+
+struct AlarmItemData {
+
+    AlarmItemData() = default;
+    AlarmItemData(const bool& everyday, const bool& active, const bool& selected, const int& day, const int& month, const int& year, const int& hours, const int& minutes)
```

Comment:
Qui direi di evitare l'uso di riferimenti costanti, usiamo direttamente i parametri per valore.
Che ne dici?

Reply:
Addirittura, se togli tutti i costruttori in C++11 la sintassi con le graffe è valida. Es:
```
struct Point { int x, y; };

void print(Point p)
{...}

int main(){
  print({0,5});
}
```